plugins/jobs/src/js/JobsTabContainer.js

```diff
@@ -0,0 +1,156 @@
+import mixin from "reactjs-mixin";
+import PropTypes from "prop-types";
+import React from "react";
+import { routerShape } from "react-router";
+import { StoreMixin } from "mesosphere-shared-reactjs";
+
+// these have to go!
+import DCOSStore from "#SRC/js/stores/DCOSStore";
+import JobTree from "#SRC/js/structs/JobTree";
+
+import JobFilterTypes from "./constants/JobFilterTypes";
+import JobsTab from "./components/JobsTab";
+import JobsTabLoading from "./components/JobsTabLoading";
+
+const METHODS_TO_BIND = [
+  "handleFilterChange",
+  "resetFilter",
+  "resetFilterQueryParams"
+];
+
+var DEFAULT_FILTER_OPTIONS = {
+  searchString: ""
+};
+
+class JobsTabContainer extends mixin(StoreMixin) {
+  constructor() {
+    super(...arguments);
+
+    this.state = { ...DEFAULT_FILTER_OPTIONS };
+
+    this.store_listeners = [
+      { name: "dcos", events: ["change"], suppressUpdate: false },
```

Comment:
Are we getting rid of the parts that we know are not needed, like this DcosStore dependency?

Reply:
Yeah, this will be part of the next PR 👍🏻